azure-pipelines.bundlesize.yml

```diff
@@ -31,16 +33,87 @@ jobs:
         displayName: 'Publish Bundle Size information to Azure Dev Ops Artifacts'
         inputs:
           PathtoPublish: 'apps/test-bundles/dist/bundlesizes.json'
+          ArtifactName: bundlesizes-react
 
       - task: PublishBuildArtifacts@1
         displayName: 'Publish Artifact dist folder upon build for debug'
         inputs:
           PathtoPublish: 'apps/test-bundles/dist'
-          ArtifactName: distdrop
+          ArtifactName: distdrop-react
+
+  - job: build-northstar
+    timeoutInMinutes: 75
+    pool:
+      vmImage: 'windows-2019'
+    steps:
+      - task: NodeTool@0
+        inputs:
+          versionSpec: '12.x'
+        displayName: 'Install Node.js'
+
+      - script: npx midgard-yarn install
+        displayName: yarn
+
+      - script: yarn build --to @fluentui/react-northstar --no-cache
+        displayName: yarn build to @fluentui/react-northstar
+
+      - script: yarn workspace test-bundles bundle:size
+        displayName: yarn bundle test-bundles
+        env:
+          PACKAGE: @fluentui/react-northstar
+
+      - script: yarn bundlesizecollect
+        displayName: 'Collate Bundle Size Information'
+
+      - task: PublishBuildArtifacts@1
+        displayName: 'Publish Bundle Size information to Azure Dev Ops Artifacts'
+        inputs:
+          PathtoPublish: 'apps/test-bundles/dist/bundlesizes.json'
+          ArtifactName: bundlesizes-northstar
+
+      - task: PublishBuildArtifacts@1
+        displayName: 'Publish Artifact dist folder upon build for debug'
+        inputs:
+          PathtoPublish: 'apps/test-bundles/dist'
+          ArtifactName: distdrop-northstar
+
+  - job: merge
+    pool:
+      vmImage: 'windows-2019'
+    dependsOn:
+      - build-react
+      - build-northstar
+    steps:
+      - task: DownloadPipelineArtifact@2
+        displayName: 'Download Pipeline Artifact Fabric'
```

Comment:
Let's not use "Fabric" in new things :) -- maybe "React" to match the name of the job?